packages/client/src/data_commons_client.ts

```diff
@@ -507,23 +574,33 @@ class DataCommonsClient {
         ) {
           const series =
             populationObservations.data[TOTAL_POPULATION_VARIABLE][entityDcid];
+          const populationFacet = _.get(
+            populationObservations.facets,
+            series.facet || "",
+            {} as StatMetadata
+          );
 
           const closestPopulationObservation = this.getClosestObservationToDate(
             series.series,
             observation.date
           );
+          const scalingFactor = facet.scalingFactor || 1;
+          const quotientValue =
+            closestPopulationObservation &&
+            !_.isEmpty(closestPopulationObservation) &&
```

Comment:
curious why you need both `closestPopulationObservation` and `!_.isEmpty(closestPopulationObservation)` checks?

Reply:
Good catch- removed `closestPopulationObservation &&`